addons/api/addon/generated/models/host.js

```diff
@@ -44,9 +44,27 @@ export default class GeneratedHostModel extends BaseModel {
   })
   version;
 
+  @attr({
+    description: 'Attributes specific to the host type',
+  })
+  attributes;
+
+  @attr('string', {
```

Comment:
I am on the fence about whether attributes fields should be included in the generated model.  However, they should probably be grouped together with other attributes fields, for readability.

Reply:
Understood. As per last time we spoke what's the difference between a simple `model` and a `generated/model` I understood `generated/model` are fields with primitive values (as `address` is).

But I see what you mean my grouping `attributes` together.

That said, IMHO we are going to start messing ember (perhaps internal) good practices with context of the app.

Will perform these changes tho.

Reply:
There are a few heuristics.  Model generation is aspirational.  Ultimately it's a question of _what can be actually be generated_ when we're ready to do it.  Right now, `attributes` can't be, because they are not present in the schema JSON.  That means `attributes` fields should probably not go in the generated model.  Let's table that for after this work though.